docs/datasets/scannet.md

```diff
@@ -0,0 +1,403 @@
+# ScanNet
+
+## Dataset preparation
+
+For the overall process, please refer to [scannet](https://github.com/open-mmlab/mmdetection3d/blob/master/data/scannet/README.md/).
```

Comment:
refer to the \[README\]\(https://github.com/open-mmlab/mmdetection3d/blob/master/data/scannet/README.md/) page for ScanNet